src/components/Modal.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from "react";
+import { TaskType as TaskType } from "../TaskType";
+import { v4 as uuidv4 } from "uuid";
+
+const Modal = ({ handleClose, addTask, isEditing, currTask, updateTask, taskType}) => {
+  const [taskName, setTaskName] = React.useState("");
+  const [state, setState] = React.useState(taskType);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (!taskName) return;
+    if(!isEditing){
+      const newTask = { name: taskName, id: uuidv4(), date: Date.now(), state };
```

Comment:
Why is there a mismatch on `id`? Here you are using `uuid()` to generate `id` for new task. But in `tasks.js` file, you have used static string values as `id`. Be consistent in whatever you use.